tests/lib/utils/utils-test.js

```diff
@@ -186,3 +186,38 @@ describe('parseArgs', () => {
     expect(parsedArgs).toEqual(['asd', 'qwe', 'zxc']);
   });
 });
+
+describe('getPropertyValue', () => {
```

Comment:
OK, I went ahead and added tests for a simple object. Can you take another look?